docs/source/virtual_ecosystem/theory/animals/animal_theory.md

```diff
@@ -23,12 +23,205 @@ language_info:
   version: 3.11.9
 ---
 
-# Theory of the animals
+# Theory of the Animal Model
+
+This page outlines the theoretical basis for the animal model within the Virtual
+Ecosystem. It details the representation of functional groups and cohorts,
+the processes governing their interactions with the environment, and their contributions
+to ecosystem-level dynamics. The page also highlights the key state variables tracked
+for each cohort and the links between animal processes and other ecosystem modules.
 
 :::{admonition} In progress 🛠️
 
-This is a high level landing page from animal theory, that will be filled out at a later
-date.
+Much of the Animal Model follows the logic of the Madingley Model (Harfoot et al,. 2014)
```

Comment:
Not urgent but would be good to add the reference to the bib file and cite by key